mullvad-vpn - anything except wireguard is broken

### Describe the bug
After enabling `services.mullvad-vpn` with `package = pkgs.mullvad-vpn` (CLI and GUI) I am unable to use any other protocol except wireguard

### Steps To Reproduce
Steps to reproduce the behavior:
1. Install `mullvad-vpn` and related service from `unstable` channel
2. Run `mullvad-gui`, switch to openvpn, try connecting

### Expected behavior
All protocols should work

### Additional context
Clearly lacks some files:
```
mullvad-daemon[63963]: [talpid_core::tunnel_state_machine::connecting_state][ERROR] Error: Failed to start tunnel
mullvad-daemon[63963]: Caused by: Failed while listening for events from the OpenVPN tunnel
mullvad-daemon[63963]: Caused by: No OpenVPN binary found at /nix/store/minlny5idf76n6fjxwgy8ln1kycz9a1x-mullvad-vpn-2022.5/bin/openvpn
```
Probably needs something similar to whatever is done in `mullvad` (CLI) package:
```bash
# Files necessary for OpenVPN tunnels to work.
''
  mkdir -p $out/share/mullvad
  cp dist-assets/ca.crt $out/share/mullvad
  ln -s ${openvpn-mullvad}/bin/openvpn $out/share/mullvad
  ln -s ${shadowsocks-rust}/bin/sslocal $out/share/mullvad
  ln -s $out/lib/libtalpid_openvpn_plugin.so $out/share/mullvad
'' +
# Set the directory where Mullvad will look for its resources by default to
# `$out/share`, so that we can avoid putting the files in `$out/bin` --
# Mullvad defaults to looking inside the directory its binary is located in
# for its resources.
''
  wrapProgram $out/bin/mullvad-daemon \
    --set-default MULLVAD_RESOURCE_DIR "$out/share/mullvad"
'';
 ```
### Notify maintainers
@ataraxiasjel @ymarkus @Br1ght0ne

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.88, NixOS, 23.05 (Stoat), 23.05.20230119.d7705c0`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.12.0`
 - nixpkgs: `/nix/store/b3g1132qzp1ns21zhrsh7cfs4sb1dql6-source`
```

--- a/pkgs/applications/networking/mullvad-vpn/default.nix
+++ b/pkgs/applications/networking/mullvad-vpn/default.nix
@@ -79,6 +79,9 @@ stdenv.mkDerivation rec {
 
     wrapProgram $out/bin/mullvad-vpn --set MULLVAD_DISABLE_UPDATE_NOTIFICATION 1
 
+    wrapProgram $out/bin/mullvad-daemon \
+        --set-default MULLVAD_RESOURCE_DIR "$out/share/mullvad/resources"
+
     sed -i "s|Exec.*$|Exec=$out/bin/mullvad-vpn $U|" $out/share/applications/mullvad-vpn.desktop
 
     runHook postInstall

